### ISSUE TITLE
Require validation when changing email

### ISSUE THREAD

**Issue by jnm** [kobotoolbox/kpi#2287]
An existing account can change its email address without validation. This sometimes causes problems with password recovery, etc.

**Comment by magicznyleszek** [comment]
@magicznyleszek There is a nice package https://www.npmjs.com/package/isemail that I will use on FE

**Comment by jnm** [comment]
I was thinking something similar to what we do when someone registers for a new account. We send a message to the address they provide, and we don't activate the new account until someone clicks the unique link sent in the validation message. It seems mostly back-end, but we do need to consider the UX. What happens when someone changes their email? Do we continue using their old address until they verify the new one? We should have some way of displaying that the new address is unverified. I know lots of other services deal with this, but to memory, most allow accounts to have multiple email addresses, which we do not.

**Comment by magicznyleszek** [comment]
Yeah, the "email with link" is much better idea than RegEx-based validation. 

I think that we should use old email until the new one is verified. And we can display a message near the email field "This email us unverified. Click here to resend the verification email. Until then we will use your old email <old email> for communication and password resetting."?

**Comment by jnm** [comment]
We should probably also send an alert to the old email address saying that someone is attempting to change (or has already changed) the address on their account.

**Comment by JacquelineMorrissette** [comment]
The email is now sent with the upgrade to Django Allauth, but sending a message to the original email address still needs to be addressed
